docs/modules/ROOT/pages/reference/extensions/sap.adoc

```diff
@@ -34,10 +34,11 @@ Check the xref:user-guide/index.adoc[User guide] for more information about writ
 endif::[]
 
 [id="extensions-sap-camel-quarkus-limitations"]
-== Camel Quarkus limitations
+== SAP Extension Camel Quarkus limitations
```

Comment:
Unfortunately, this wont work. The heading is fixed as the this part is automatically generated by some tooling in the project.